conftest.py

```diff
@@ -184,15 +184,22 @@ def sr_disk_wiped(host, sr_disk):
     yield sr_disk
 
 @pytest.fixture(scope='session')
-def sr_disk_for_all_hosts(host, sr_disk):
+def sr_disk_for_all_hosts(host):
+    master_disks = host.available_disks()
+    assert len(master_disks) > 0, "a free disk device is required on the master host"
```

Comment:
the assert can be put in `available_disks` perhaps with a `raise` flag. To avoir duplicating the assert and error message.

Reply:
I'm not sure it's a good idea: `available_disks` is clear, it returns the available disks, IMHO an assert in this helper is equal to a bad naming.

Reply:
Also `available_disks` is used in the for loop after this line without `assert`. ;)

Reply:
If there's no consensus, then I guess I won't change it :D

Reply:
Tht's why I proposed a flag to param it ;)

Reply:
Sometimes, it's not a problem to duplicate a line, if it's not abusive. ^^"